tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_report.py

```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+from marionette_driver import expected, By, Wait
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.base import Base
+from gaiatest.apps.messages.app import Messages
+
+
+class TestSmsReport(GaiaTestCase):
+
+    time_sent = time.time()
+
+    def test_sms_report(self):
+        """This test sends a text message to itself. It waits for a reply message.
+           Then it checks the info in the report.
+        """
+        _text_type = (By.CSS_SELECTOR, 'p[data-l10n-id="message-type-sms"]')
+        _sender = (By.CSS_SELECTOR, 'a[data-name]')
+        _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
+
+        
+        # Configuring the test
```

Comment:
If you want to configure the test, there's a python method called `setUp()`. You can see an example here: https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_bookmark.py#L16